azurerm/internal/services/sql/resource_arm_sql_database.go

```diff
@@ -318,6 +318,12 @@ func resourceArmSqlDatabase() *schema.Resource {
 				Default:  false,
 			},
 
+			"zone_redundant": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
```

Comment:
since we're shipping this in a major version of the provider we can make this Optional to force users to select it if required

```suggestion
```